go/ct/driver/regressions.go

```diff
@@ -117,6 +117,12 @@ func doRegressionTests(context *cli.Context) error {
 			expected := state.Clone()
 			rule.Effect.Apply(expected)
 
+			// TODO: do not only skip state but change 'pc_on_data_is_ignored' rule to anyEffect
```

Comment:
Is there an issue number for this that we can add to this TODO?

Reply:
I created an issue and added it to the comments.